File: MOBILE/app_sintec/lib/shared/model/retorno.dart
class Retorno{
  int? id;
  int idUsuario;
  int idServico;
  int? idOcorrencia;
  String dataExecucao;
  String longitude;
  String latitude;
  int flagRacp;
  String modeloAparelho;
  String marcaAparelho;
  int sincronizado;
  int ativo;

  Retorno({
    this.id,
    required this.idUsuario,
    required this.idServico,
     this.idOcorrencia,
    required this.dataExecucao,
    required this.longitude,
    required this.latitude,
    required this.flagRacp,
    required this.modeloAparelho,
    required this.marcaAparelho,
    required this.sincronizado,
    required this.ativo
  });
  factory Retorno.fromSQLite(Map map) {
    return Retorno(
      id: map["id"],
      idUsuario: map["id_usuario"],
      idServico: map["id_servico"],
      idOcorrencia: map["id_ocorrencia"],
      dataExecucao: map["data_execucao"],
      longitude: map["longitude"],
      latitude: map["latitude"],
      flagRacp: map["flag_racp"],
      modeloAparelho: map["modelo_aparelho"],
      marcaAparelho: map["marca_aparelho"],
      sincronizado: map["sincronizado"],
      ativo: map["ativo"]
    );
  }

  static List<Retorno> fromSQLiteList(List<Map> listMap) {
    List<Retorno> lista = [];
    for (Map item in listMap) {
      lista.add(Retorno.fromSQLite(item));
    }
    return lista;
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['idUsuario'] = this.idUsuario;
    data['idServico'] = this.idServico;
    data['idOcorrencia'] = idOcorrencia ;
    data['dataExecucao'] = this.dataExecucao;
    data['longitude'] = this.longitude;
    data['latitude'] = this.latitude;
    data['flagRacp'] = this.flagRacp;
    data['modeloAparelho'] = this.modeloAparelho;
    data['marcaAparelho'] = this.marcaAparelho;
    data['sincronizado'] = this.sincronizado;
    data['ativo'] = this.ativo;
    return data;
  }


}

File: api-sintec/src/main/java/br/com/sintec/model/Retorno.java
package br.com.sintec.model;

import java.time.LocalDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@ToString
@Getter
@Setter
@Entity
@Table(name="SERV_RETORNO")
public class Retorno {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@EqualsAndHashCode.Include
	@Column(name = "ID")
	private Long id;
	@ManyToOne
	@JoinColumn(name="ID_USUARIO")
	private Usuario usuario;
	@ManyToOne
	@JoinColumn(name="ID_SERVICO")
	private Servico servico;
	@ManyToOne
	@JoinColumn(name="ID_OCORRENCIA")
	private Ocorrencia ocorrencia;
	@Column(name = "DT_EXECUCAO")
	private LocalDateTime dataExecucao;
	@Column(name = "CD_LATITUDE")
	private String latitude;
	@Column(name = "CD_LONGITUDE")
	private String longitude;
	@Column(name="NM_MODELO_APARELHO")
	private String modeloAparelho;
	@Column(name="NM_MARCA_APARELHO")
	private String marcaAparelho;
	@Column(name="FLAG_RACP")
	private int flagRacp;
	@Column(name = "ST_ATIVO")
	private int ativo;
}
